Drop requirements that are needed only for testenvs we do not run.

--- wcwidth-0.2.10/requirements-tests39.in.orig
+++ wcwidth-0.2.10/requirements-tests39.in
@@ -1,15 +1,7 @@
 # For Python 3.9 *and newer*
 pytest>=7.4.2
 pytest-cov>=4.1.0
-flake8
-autopep8
-docformatter
-isort
-pydocstyle
-restructuredtext_lint
-doc8
 pygments
-pylint>=2.15
 astroid>=2.11
 wrapt>=1.14
 pytest-benchmark
